service/email/src/main/java/com/ttbmp/cinehub/service/email/mock/MockEmailService.java

```diff
@@ -0,0 +1,16 @@
+package com.ttbmp.cinehub.service.email.mock;
+
+
+import com.ttbmp.cinehub.core.service.email.EmailService;
+import com.ttbmp.cinehub.core.service.email.EmailServiceRequest;
+
+/**
+ * @author Palmieri Ivan
+ */
+public class MockEmailService implements EmailService {
+
+    @Override
+    public void sendMail(EmailServiceRequest emailServiceRequest) {
+        //Empty section
```

Comment:
Why this method is empty?

Reply:
Because there is already a real implementation of the method in the SendEmailService class, I should delete the Mock later

Reply:
Mock are useful for testing purpose, you should declare them so that they would emulate the real behaviour and you shouldn't delete them